### PR TITLE
Swagger SDK for HDInsight

### PR DESCRIPTION
**Pull request by shefaliv**
Sending a preliminary PR for the HDI move to Swagger. This just contains the management project (generated + customizations). Hopefully this unblocks the Json updates. I'll send another PR with tests and any necessary changes!

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/HDInsight/Management.HDInsight/Management.HDInsight.csproj [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.HDInsight</PackageId>
+    <Description>Azure HDInsight Management SDK Library</Description>
+    <AssemblyName>Microsoft.Azure.Management.HDInsight</AssemblyName>
+    <VersionPrefix>3.0.0-preview</VersionPrefix>
+    <PackageTags>Microsoft Azure HDInsight Management;HDInsight;HDInsight Management</PackageTags>    
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
```
Comment (shahabhijeet) [inline src/SDKs/HDInsight/Management.HDInsight/Management.HDInsight.csproj]: @shefaliv the only supported target frameworks are net452 and netstandard1.4.
We do not support .NET Core as of now.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/HDInsight/Management.HDInsight/Management.HDInsight.csproj (hunk 1 above)
Comment (shahabhijeet) [inline src/SDKs/HDInsight/Management.HDInsight/Management.HDInsight.csproj]: @shefaliv please add link to the corresponding swagger PR

Comment (shefaliv) [inline src/SDKs/HDInsight/Management.HDInsight/Management.HDInsight.csproj]: ah okay. I thought this is what the documentation said to do. Let me fix it!

**Comment by shefaliv** [comment]
Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/1312

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Need test coverage before we can accept this PR.
Please use the latest nugets (including latest resource manager) to record all your tests.

**Comment by shahabhijeet** [comment]
@shefaliv add "needs-review" when you have addressed the PR.

**Comment by shahabhijeet** [comment]
@shefaliv please either reopen this PR or create a new one, once you are ready to merge your new SDK.

**Pull request closed without merging**